Source/WebKit/Shared/API/Cocoa/WKRemoteObjectCoder.mm

```diff
@@ -91,10 +92,10 @@ - (BOOL)validateClassSupportsSecureCoding:(Class)objectClass;
 @end
 
 @implementation WKRemoteObjectEncoder {
-    RefPtr<API::Dictionary> _rootDictionary;
-    API::Array* _objectStream;
+    const RefPtr<API::Dictionary> _rootDictionary;
```

Comment:
I’m surprised this works in an Objective-C object. It’s sort of amazing that `lazyInitialize` is powerful enough to do this even after an early return from the `init` method. Also slightly surprised it’s a `RefPtr` and not a `Ref`.

Reply:
Ref wouldn't work with Objective C and init functions because Ref doesn't have a default constructor.